public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -173,13 +173,34 @@ export class DashboardModel implements TimeModel {
     if (options?.autoMigrateOldPanels || !config.angularSupportEnabled || config.featureToggles.autoMigrateOldPanels) {
```

Comment:
hmm, so you have to enable both `autoMigrateOldPanels` AND have `autoMigrateGraphPanel`?

i guess this is to disable migrating graph but still migrate others? is that a realistic scenario if we're gonna tackle graph first?

logic might be less delicate if autoMigrateOldPanels did all and autoMigrateGraphPanel just did Graph?

Reply:
Currently logic is that you can enable either `autoMigrateOldPanels` or `autoMigrateGraphPanel` to have `graph` auto migrated to `timeseries`

see [these lines](https://github.com/grafana/grafana/pull/79369/files#diff-1246109f7d7d3f52aa2e377c19ae29439b007cac8424e2cdbc8487aca3d4969dR177-R181) that skip over graph in original check but then is addressed in later check (which we will move the other panels over to eventually as well and remove current check here

Kinda convoluted but gets us to state where `autoMigrateOldPanels` functionality is the same as before and we can incrementally add panel specific feature toggles for migration

Reply:
For G11 Angular will be disabled with no option to have it enabled 😬 

Purpose of these feature toggles is to provide some options for us to help users migrate their deprecated panels to the new versions in a more granular fashion (i.e. starting out with just graph -> timeseries, but will follow-up with a few more major panels, probably table, piechart, and worldmap)